src/vm/comutilnative.cpp

```diff
@@ -1524,7 +1528,11 @@ FCIMPL5(VOID, Buffer::InternalBlockCopy, ArrayBase *src, int srcOffset, ArrayBas
     _ASSERTE(count >= 0);
 
     // Copy the data.
+#if defined(_WIN64)  
+    JIT_MemCpy(dst->GetDataPtr() + dstOffset, src->GetDataPtr() + srcOffset, count);
+#else    
```

Comment:
Or ideally ... fixed in the stock memmove implementation, so that all programs benefit from it and we do not have to have a private copy that has to be kept up to date with the latest tweaks.